spyder/plugins/console/widgets/shell.py

```diff
@@ -982,7 +992,7 @@ class TerminalWidget(ShellBaseWidget):
     COM = 'rem' if os.name == 'nt' else '#'
     INITHISTORY = ['%s *** Spyder Terminal History Log ***' % COM, COM,]
     SEPARATOR = '%s%s ---(%s)---' % (os.linesep*2, COM, time.ctime())
-    go_to_error = Signal(str)
+    sig_go_to_error_requested = Signal(str)

```

Comment:
Please add a comment above this signal to describe what it does.

Reply:
Fixed